pyproject.toml

```diff
@@ -0,0 +1,113 @@
+[build-system]
+requires = ["hatchling", "hatch-fancy-pypi-readme"]
+build-backend = "hatchling.build"
+
+[project]
+name = "gpio"
+dynamic = ["version", "readme"]
+description = "gpio access via the standard linux sysfs interface"
+license = {file = "LICENSE"}
+requires-python = ">= 3.7"
+authors = [
+    { name = "Philip Howard", email = "phil@pimoroni.com" },
+    { name = "Garrett Berg", email = "vitiral@gmail.com" },
+]
+maintainers = [
+    { name = "Philip Howard", email = "phil@pimoroni.com" },
```

Comment:
I'm still reviewing CL's so I think I can be added here no?

Reply:
Will make the change, it’s pulled over from the boilerplate so it’s more an oversight than a deliberate attempt to erase you 🤣

Reply:
Fixed!